stdlib/REPL/src/REPLCompletions.jl

```diff
@@ -165,14 +165,14 @@ function complete_symbol(sym::String, ffunc, context_module::Module=Main)
         end
         # Looking for a binding in a module
         if mod == context_module
-            # Also look in modules we got through `using`
-            mods = ccall(:jl_module_usings, Any, (Any,), context_module)::Vector
-            for m in mods
-                append!(suggestions, filtered_mod_names(p, m::Module, name))
-            end
-            append!(suggestions, filtered_mod_names(p, mod, name, true, true))
-        else
-            append!(suggestions, filtered_mod_names(p, mod, name, true, false))
+            # special case `Core` and `Base` bindings
```

Comment:
Do these not show up via usings=true normally? I guess I don't really know how names works